### PR TITLE
Update base OS from RHEL8 -> RHEL9

### PR DESCRIPTION
**Pull request by kyguy**

### CODE REVIEW

**Review by MikeEdgar — COMMENTED** [review]
When making the changes for console, I noticed these packages being removed do not seem to be valid for RHEL 9. It doesn't cause the removal to fail, but perhaps there are other packages that should be listed and removed in RHEL 9 that we're unaware of.
```
No match for argument: python3-libxml2
No match for argument: dbus-glib
No match for argument: python3-dmidecode
No match for argument: python3-syspurpose
No match for argument: dnf-plugin-subscription-manager
No match for argument: None
```

**Comment by k-wall** [comment]
@OwenCorrigan76 would you be able to rework the removal list?

File: proxy/image.yaml [hunk 1]
```diff
@@ -32,10 +32,10 @@ packages:
   content_sets:
     x86_64:
       # Required for tini
-      - amq-streams-2-for-rhel-8-rpms
+      - amq-streams-2-for-rhel-9-rpms
       # Required for base image
-      - rhel-8-for-x86_64-baseos-eus-rpms__8_DOT_6
-      - rhel-8-for-x86_64-appstream-eus-rpms__8_DOT_6
+      - rhel-9-for-x86_64-baseos-eus-rpms__9_DOT_2
+      - rhel-9-for-x86_64-appstream-eus-rpms__9_DOT_2
```
Comment (k-wall) [inline proxy/image.yaml:38]: The line below "nss-3.67.0-7.el8_5 # experiment - try pinning to the FIPS validated nss." needs to be removed.  This related to RHEL 8.6.

Comment (OwenCorrigan76) [reply proxy/image.yaml:38]: @k-wall @MikeEdgar I've double checked and those packages are not present in the RHE-9 image.  I'll remove above packages from removal list.

Comment (k-wall) [reply proxy/image.yaml:38]: Is there anything coming from RHEL 9 base image that is superfluous to us?  If so, we should remove it/them

**Comment by kyguy** [comment]
Haven't had time today to do further analysis or testing on the RPM removal myself but I think we covered a good chunk of RPMs. If anyone finds anything else before we  are unblocked by the `tini` RPM release feel free to update the PR. If we have time tomorrow morning I'll take a closer look, otherwise, I'll move forward with the CR2 builds to unblock the release.

It's worth noting that we can always remove RPMs post CR2 and even GA and pick up the changes in the next respin release!

**Pull request merged**